contracts/USM.sol

```diff
@@ -12,11 +12,9 @@ import "./IOracle.sol";
 contract USM is ERC20 {
     using SafeMath for uint;
 
-    // 0.1% (10 parts per 10,000)
-    uint constant MINT_FEE = 10;
-    // 0.5%; (50 parts per 10,000)
-    uint constant BURN_FEE = 50;
-    uint constant PERCENTAGE_BASE = 10000;
+    uint constant UNIT = 1000000000000000000000000000; // 10**27, MakerDAO's RAY
```

Comment:
It's best to use a single unit for decimal numbers across the project. Remember that token amounts are actually integers (wei) even if the ERC20 standard has a `decimals()` variable for convenience.